Engine/source/platform/platformNet.cpp

```diff
@@ -823,13 +823,16 @@ NetSocket Net::openConnectTo(const char *addressString)
       if (socketFd != InvalidSocketHandle)
       {
          setBlocking(handleFd, false);
-         if (::connect(socketFd, (struct sockaddr *)&ipAddr, sizeof(ipAddr)) == -1 &&
-            errno != EINPROGRESS)
+         if (::connect(socketFd, (struct sockaddr *)&ipAddr, sizeof(ipAddr)) == -1)
```

Comment:
Keep in mind this code also needs to work for unix/bsd sockets api. Also edge cases can crop up, etc - which is why I was being cautious here. ;)

Taken from the man page on OSX:

	Upon successful completion, a value of 0 is returned.  Otherwise, a value of -1 is returned and the
	     global integer variable errno is set to indicate the error.

	ERRORS
	     The connect() system call will fail if:

	     [EINPROGRESS]      The socket is non-blocking and the connection cannot be completed immediately.  It
	                        is possible to select(2) for completion by selecting the socket for writing.

Which would imply ::connect == 0 or ::connect == -1 and setting errno to EINPROGRESS are the two valid success conditions here.

Thankfully reading through this code, I realize now that it should fall past this block if those conditions are met so this stuff should be ok. Good work 👍